src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,157 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int INIT_SIZE = 0;
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    MyLinkedList() {
+        size = INIT_SIZE;
```

Comment:
We don't need this constructor, `size` variable will be initialized by `0` by default